paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/PaymentSheetAnalyticsListener.kt

```diff
@@ -83,6 +83,10 @@ internal class PaymentSheetAnalyticsListener(
                 reportFormShown(currentPaymentMethodTypeProvider())
                 eventReporter.onShowNewPaymentOptionForm()
             }
+            is PaymentSheetScreen.CvcRecollection -> {
```

Comment:
you can probably add this screen to the first branch, which has a bunch of screens that we don't add analytics for